tls/s2n_prf.c

```diff
@@ -44,6 +44,11 @@ static int s2n_sslv3_prf(struct s2n_connection *conn, struct s2n_blob *secret, s
     POSIX_ENSURE_REF(conn->handshake.hashes);
     struct s2n_hash_state *workspace = &conn->handshake.hashes->hash_workspace;
 
+    /* FIPS specifically allows MD5 for the legacy PRF */
```

Comment:
Can we cite FIPS here? I feel like the answer is probably no, but I would love a spec comment here if possible.